docs/quickstart.md

```diff
@@ -0,0 +1,164 @@
+
+    @app.route('/')
+    def hello():
+        redis.incr('hits')
+        return 'Hello World! I have been seen %s times.' % redis.get('hits')
+
+    if __name__ == "__main__":
+        app.run(host="0.0.0.0", debug=True)
+
+Next, define the Python dependencies in a file called `requirements.txt`:
+
+    flask
+    redis
+
+### Create a Docker image
+
+Now, create a Docker image containing all of your app's dependencies. You
+specify how to build the image using a file called
+[`Dockerfile`](http://docs.docker.com/reference/builder/):
+
+    FROM python:2.7
+    ADD . /code
+    WORKDIR /code
+    RUN pip install -r requirements.txt
+    CMD python app.py
+
+This tells Docker to:
+
+* Build an image starting with the Python 2.7 image.
+* Add the current directory `.` into the path `/code` in the image.
+* Set the working directory to `/code`.
+* Install the Python dependencies.
+* Set the default command for the container to `python app.py`
+
+For more information on how to write Dockerfiles, see the [Docker user guide](https://docs.docker.com/userguide/dockerimages/#building-an-image-from-a-dockerfile) and the [Dockerfile reference](http://docs.docker.com/reference/builder/).
+
+You can build the image by running `docker build -t web .`.
+
+### Define services
+
+Next, define a set of services using `docker-compose.yml`:
+
+    web:
+      build: .
+      ports:
+       - "5000:5000"
+      volumes:
+       - .:/code
+      links:
+       - redis
+    redis:
+      image: redis
+
+This template defines two services, `web` and `redis`. The `web` service:
+
+* Builds from the `Dockerfile` in the current directory.
+* Forwards the exposed port 5000 on the container to port 5000 on the host machine.
+* Mounts the current directory on the host to `/code` inside the container allowing you to modify the code without having to rebuild the image.
+* Links the web container to the Redis service.
+
+The `redis` service uses the latest public [Redis](https://registry.hub.docker.com/_/redis/) image pulled from the Docker Hub registry.
+
+### Build and run your app with Compose
+
+Now, when you run `docker-compose up`, Compose will pull a Redis image, build an image for your code, and start everything up:
+
+    $ docker-compose up
+    Pulling image redis...
+    Building web...
+    Starting composetest_redis_1...
+    Starting composetest_web_1...
+    redis_1 | [8] 02 Jan 18:43:35.576 # Server started, Redis version 2.8.3
+    web_1   |  * Running on http://0.0.0.0:5000/
+    web_1   |  * Restarting with stat
+
+If you're using [Docker Machine](https://docs.docker.com/machine), then `docker-machine ip MACHINE_VM` will tell you its address and you can open `http://MACHINE_VM_IP:5000` in a browser.
+
+If you're using Docker on Linux natively, then the web app should now be listening on port 5000 on your Docker daemon host. If http://0.0.0.0:5000 doesn't resolve, you can also try http://localhost:5000.
+
+You should get a message in your browser saying:
```

Comment:
Drop the "should" in this section. We're confident :metal: it just works, right? :smile: